### PR TITLE
msys2.msys2 version 20210604

### PR DESCRIPTION
**Pull request by Schatteneis**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/16825)

### CODE REVIEW

**Comment by Schatteneis** [comment]
dont know how to add a silent switch, i do know that you can pass "in --confirm-command --accept-messages --root C:/msys64" to the msys installer for a silent install

**Comment by denelon** [comment]
@Schatteneis the wingetcreate tool is very "preview" right now. I see it doesn't prompt for the switches with this particular installer. You might be better off creating a "Package Request" issue. You would need to be familiar with modifying the manifest to add the switches and potentially some of the other meta-data in the schema.

**Comment by denelon** [comment]
If you are willing to "hand edit", you could search for "Silent" in other PRs or in the source. That should show several examples of how the switches are added to manifests.

File: manifests/m/msys2/msys2/4.0.1/msys2.msys2.yaml [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# Created using wingetcreate
+
+PackageIdentifier: msys2.msys2
+PackageVersion: 4.0.1
```
Comment (ItzLevvie) [inline manifests/m/msys2/msys2/4.0.1/msys2.msys2.yaml:4]:
```suggestion
PackageVersion: 20210604
```

File: manifests/m/msys2/msys2/4.0.1/msys2.msys2.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: msys2.msys2
+PackageVersion: 4.0.1
+PackageLocale: en-US
+Publisher: MSYS2
```
Comment (ItzLevvie) [inline manifests/m/msys2/msys2/4.0.1/msys2.msys2.locale.en-US.yaml:6]:
```suggestion
Publisher: The MSYS2 Developers
```

File: manifests/m/msys2/msys2/4.0.1/msys2.msys2.locale.en-US.yaml [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Created using wingetcreate
+
+PackageIdentifier: msys2.msys2
+PackageVersion: 4.0.1
+PackageLocale: en-US
+Publisher: MSYS2
+PackageName: MSYS2
+License: BSD 3-Clause
+ShortDescription: Installer for minimal system 2
```
Comment (ItzLevvie) [inline manifests/m/msys2/msys2/4.0.1/msys2.msys2.locale.en-US.yaml:9]:
```suggestion
ShortDescription: MSYS2 is a collection of tools and libraries providing you with an easy-to-use environment for building, installing and running native Windows software.
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]
Please see my review. You'll also need to update the version folder.

**Comment by Schatteneis** [comment]
@ItzLevvie how do i update the version folder? iam very new to git

**Comment by ItzLevvie** [comment]
1) Download the GitHub Desktop client and login with your details
2) Clone your winget-pkgs fork
3) Choose `autogenerated/msys2.msys2/b0dedb6b-2368-43dd-aac6-d5be96d368c8` by clicking on branch
4) Open up File Explorer then navigate to manifests/m/msys2/msys2 and make your changes there
5) Write a commit summary and then click on the Push changes to branch button.

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**